### DISCUSSION TITLE
incorrectly wire up controller

### DISCUSSION THREAD

**Discussion by rajagopal1997** [react-hook-form/react-hook-form#8207] [Q&A]
### Version Number

7.27.1

### Codesandbox/Expo snack

https://codesandbox.io/s/reacthookformsforsetfocus123-2yuzjy?file=/src/App.js

### Steps to reproduce

1. Go to (https://codesandbox.io/s/reacthookformsforsetfocus123-2yuzjy?file=/src/App.js)
2. I'm taking React Table with the version of 7.7.0
3. With that react table i'm adding react-hook-from (version 7.27.1)
4. After Adding these two libraries TABINDEX (Tab) button not working

### Expected behaviour

After adding these two libraries, i'm getting this output ( **https://codesandbox.io/s/reacthookformsforsetfocus123-2yuzjy?file=/src/App.js** ) in that table **TABINDEX (Tab) Button** not working for moving one cell to another cell.

i'm trying to solve this issue for last 3 weeks, but not solved and my self soo much disappointed..

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
This is not an issue to do with react hook form, we don't dictate the tab index on each of your inputs.

**Comment by rajagopal1997** [comment]
After using of React hook form useFieldArray hook tab index button was not working...before that it's working for tabindex button

**Comment by rajagopal1997** [comment]
when i remove useFieldArray hook, its working good

**Comment by bluebill1049** [comment]
`Controller` is not wired, see below

```tsx
return (
  <Controller
    render={({
      field: { name, ref },
      fieldState: { invalid, isTouched, isDirty, error }
    }) => (
      <TextField
        value={value} // ❌ 
        onChange={onChange} // ❌ 
        onBlur={onBlur} // ❌ 
        inputRef={ref} 
      />
    )}
    name={`uoms.${index}.${id}`}
    control={control}
    // rules={{ required: true }}
  />
);
```

![Screen Shot 2022-04-18 at 6 01 12 pm](https://user-images.githubusercontent.com/10513364/163777037-1d37de04-01b1-4b35-b9c6-f08672458129.png)

https://user-images.githubusercontent.com/10513364/163777333-c6285639-1364-4ae5-9638-da717182cca8.mov
